src/layer/EnvisatMerisEOCloudLayer.ts

```diff
@@ -23,4 +23,14 @@ export class EnvisatMerisEOCloudLayer extends AbstractSentinelHubV1OrV2Layer {
       description,
     });
   }
+
+  protected getTileLinks(tile: Record<string, any>): Link[] {
+    return [
+      {
+        href: tile.pathFragment,
+        rel: 'self',
+        title: 'EOCloudPath',
```

Comment:
I'm a bit torn here... On the one hand, it would be nice to use part of STAC definition here, but on the other side it is not really fitting... `rel` will always be `self`, and we can't use `title` as ID. We will need to know about the different link types though.

Maybe we can define `Link` like this?
```
{
  href: tile.pathFragment,
  type: 'eocloud',
}
```

Additionally, I assume that `href` always contains an absolute URL? If not, I think a better name would be `target`, `path`, `link` or similar...

Reply:
It was defined as:
```
{
  target: string,
  type: LinkType,
}
```